docs/getting_started/overview_ieam.md

```diff
@@ -0,0 +1,62 @@
+---
+
+copyright:
+years: 2021
+lastupdated: "2021-02-20"
+
+---
+
+{:new_window: target="blank"}
+{:shortdesc: .shortdesc}
+{:screen: .screen}
+{:codeblock: .codeblock}
+{:pre: .pre}
+{:child: .link .ulchildlink}
+{:childlinks: .ullinks}
+
+# Overview of {{site.data.keyword.edge_notm}}
+{: #overviewofedge}
+
+This section provides an overview of {{site.data.keyword.edge_notm}} ({{site.data.keyword.ieam}}).
```

Comment:
In the preview, both of these variables resolve to the full term - Open Horizon (Open Horizon).

Reply:
I changed the property from `ieam` to `edge_abbr`.